lib/Swagger/src/Discussion/Api/InternalNotificationApi.php

```diff
@@ -0,0 +1,199 @@
+
+    /**
+     * Get API client
+     *
+     * @return \Swagger\Client\ApiClient get the API client
+     */
+    public function getApiClient()
+    {
+        return $this->apiClient;
+    }
+
+    /**
+     * Set the API client
+     *
+     * @param \Swagger\Client\ApiClient $apiClient set the API client
+     *
+     * @return InternalNotificationApi
+     */
+    public function setApiClient(\Swagger\Client\ApiClient $apiClient)
+    {
+        $this->apiClient = $apiClient;
+        return $this;
+    }
+
+    /**
+     * Operation requestNotification
+     *
+     * Request notification
+     *
+     * @param string $x_wikia_internal_request  (required)
+     * @param \Swagger\Client\Discussion\Models\RequestNotificationRequest $body  (optional)
+     * @return void
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function requestNotification($x_wikia_internal_request, $body = null)
+    {
+        list($response) = $this->requestNotificationWithHttpInfo($x_wikia_internal_request, $body);
+        return $response;
+    }
+
+    /**
+     * Operation requestNotificationWithHttpInfo
+     *
+     * Request notification
+     *
+     * @param string $x_wikia_internal_request  (required)
+     * @param \Swagger\Client\Discussion\Models\RequestNotificationRequest $body  (optional)
+     * @return Array of null, HTTP status code, HTTP response headers (array of strings)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function requestNotificationWithHttpInfo($x_wikia_internal_request, $body = null)
+    {
+        // verify the required parameter 'x_wikia_internal_request' is set
+        if ($x_wikia_internal_request === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $x_wikia_internal_request when calling requestNotification');
+        }
+        // parse inputs
+        $resourcePath = "/request-notification";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('application/json'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array());
+
+        // header params
+        if ($x_wikia_internal_request !== null) {
+            $headerParams['X-Wikia-Internal-Request'] = $this->apiClient->getSerializer()->toHeaderValue($x_wikia_internal_request);
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
+
+        // body params
+        $_tempBody = null;
+        if (isset($body)) {
+            $_tempBody = $body;
+        }
+
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)